### PR TITLE
Add new words to wordlist

### PR DESCRIPTION
**Pull request by samjwu**
for ROCm documentation

https://github.com/RadeonOpenCompute/ROCm/pull/2078

### CODE REVIEW

File: .wordlist.txt [hunk 1]
```diff
@@ -62,6 +63,7 @@ enablement
 epilog
 EPYC
 ESXi
+Ethernet
```
Comment (Naraenda) [inline .wordlist.txt]:
```suggestion
ethernet
```

File: .wordlist.txt [hunk 2]
```diff
@@ -153,6 +157,7 @@ prerequisites
 PRNG
 profiler
 PyPi
+Pytorch
```
Comment (Naraenda) [inline .wordlist.txt]:
```suggestion
PyTorch
```

File: .wordlist.txt [hunk 3]
```diff
@@ -2,6 +2,7 @@ ABI
 activations
 AlexNet
 AMD
+amdclang
```
Comment (Naraenda) [inline .wordlist.txt]: See: https://github.com/RadeonOpenCompute/ROCm/pull/2078#issuecomment-1520459111

File: .wordlist.txt [hunk 4]
```diff
@@ -85,6 +87,8 @@ HBM
 heterogenous
 hipBLAS
 hipBLASLt
+hipcc
+HIPCC
```
Comment (Naraenda) [inline .wordlist.txt]: See: https://github.com/RadeonOpenCompute/ROCm/pull/2078#issuecomment-1520459111

HIPCC, in full capitalization is okay, IMHO.

**Review by Naraenda — CHANGES_REQUESTED** [review]

File: .wordlist.txt (hunk 2 above)
Comment (samjwu) [reply .wordlist.txt]: Thanks for pointing that out. Fixed.

File: .wordlist.txt (hunk 3 above)
Comment (samjwu) [reply .wordlist.txt]: Removed this word in https://github.com/RadeonOpenCompute/rocm-docs-core/pull/120/commits/7db8a5fbfad1beb6107a45138696961f09a0853a

File: .wordlist.txt (hunk 4 above)
Comment (samjwu) [reply .wordlist.txt]: Done. Removed hipcc and kept HIPCC

File: .wordlist.txt (hunk 1 above)
Comment (samjwu) [reply .wordlist.txt]: Done in https://github.com/RadeonOpenCompute/rocm-docs-core/pull/120/commits/e862e62fb9b8669bfbd4671672c3a795b9486698

**Comment by samjwu** [comment]
> LGTM! Apologies for all the tiny nits smile

No need to apologize! They were helpful for checking spelling.

**Pull request merged**